modules/system/traits/AssetMaker.php

```diff
@@ -142,7 +146,11 @@ public function addJs($name, $attributes = [])
     public function addCss($name, $attributes = [])
     {
         if (is_array($name)) {
-            $name = $this->combineAssets($name);
+	        $assetPath = $this->assetPath;
+	        if (!starts_with($assetPath, [base_path()])) {
```

Comment:
Can you put this in a helper method, something like `getLocalPath()`?

Reply:
Sure thing!